### PR TITLE
Rename method to match conventions

### PR DESCRIPTION
**Pull request by W-lfchen**

### CODE REVIEW

**Comment by W-lfchen** [comment]
The StudentLinks adjustment is still needed

**Comment by W-lfchen** [comment]
This is most likely not ideal, but it works

File: src/graderPublic/java/h13/util/StudentLinks.java [hunk 1]
```diff
@@ -471,7 +471,14 @@ public enum GameSceneMethodLink implements ClassMethodLink {
...
-                () -> GameScene.class.getDeclaredMethod("initGameboard")
+                () -> {
+                    // Typo in initial student template
+                    try {
+                        return GameScene.class.getDeclaredMethod("initGameboard");
+                    } catch (NoSuchMethodException e) {
+                        return GameScene.class.getDeclaredMethod("initGameBoard");
+                    }
+                }
             ))),
```
Comment (Rdeisenroth) [inline src/graderPublic/java/h13/util/StudentLinks.java:482]: This is not very clean and will certainly lead to confusion as it does not provide proper Error Messages if students tamper with the method signature.
Please use the following snippet instead:
```java
INIT_GAMEBOARD_METHOD(Assertions3.assertMethodExists(
    BasicTypeLink.of(GameScene.class),
    MatcherUtils.or(
        identical("initGameboard"),
        identical("initGameBoard")
    )
)),
```

**Review by Rdeisenroth — CHANGES_REQUESTED** [review]
Please cleanup the changes in StudentLinks

File: src/graderPublic/java/h13/util/StudentLinks.java (hunk 1 above)
Comment (W-lfchen) [reply src/graderPublic/java/h13/util/StudentLinks.java]: Should be implemented now.

**Review by Rdeisenroth — APPROVED** [review]

**Pull request merged**